core/templates/dev/head/css/oppia.css

```diff
@@ -3989,6 +3989,16 @@ md-card.preview-conversation-skin-supplemental-card {
   margin-right: 2px;
   vertical-align: initial;
 }
+/* Accessibility workaround for icon buttons and links
+  Taken from:
+  https://www.nczonline.net/blog/2013/04/01/making-accessible-icon-buttons/ */
+.oppia-icon-btn-label {
```

Comment:
Perhaps we should call this something like .oppia-accessibility-label? Otherwise there will be questions about what the "btn label" is for and people may or may not use it with new buttons.

That said, please modify the proposed class name accordingly if this is meant to be scoped only to buttons, or to icons, or both. Thanks!